lib/sql_footprint/sql_statements.rb

```diff
@@ -3,12 +3,17 @@
 
 module SqlFootprint
   class SqlStatements
+    NEWLINE = "\n".freeze
     def initialize
       @statements = Set.new
     end
 
     delegate :to_a, :add, :sort, :count, to: :statements
 
+    def to_s
+      statements.sort.join(NEWLINE) + NEWLINE
```

Comment:
@brandonjoyce I think this code belongs in the `SqlCapturer` class because it's a dependency to writing these statements to a file rather than representing the collection as a string.